App.Cards/Repositories/InMemoryCardsRepository.cs

```diff
@@ -16,29 +17,16 @@ public interface ICardsRepository
     }
     public class InMemoryCardsRepository : ICardsRepository, ITransientDependency
     {
-        private static readonly List<Card> cards = Init();
+        private AppCardsDbContext context;
```

Comment:
This is invalid. You should create new class for EF repository implementation. The name does not correspond to class behaviour